Framework/Kernel/inc/MantidKernel/EnumeratedStringProperty.h

```diff
@@ -0,0 +1,106 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2007 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+#include "MantidKernel/EnumeratedString.h"
+#include "MantidKernel/Exception.h"
```

Comment:
Is this import used?

Reply:
```Exception.h``` removed

Reply:
It's still right here. Did you forget to push a commit?

Reply:
Oh, I did  not pushed it yet..